content/content.js

```diff
@@ -221,7 +234,20 @@ var getTranslation = async (image, coordinates, api, idToken, source_lang, targe
     if (response.error) {
       showTranslationDialog(`Error: translation is not valid: ${response.error}`, coordinates, "", undefined, overlayId)
     } else if (response.translation) {
-      showTranslationDialog(response.translation, coordinates, response.original, response.pronunciation, overlayId)
+      showTranslationDialog(response.translation + "\n\n retrieving in-depth translation", coordinates, response.original, response.pronunciation, overlayId)
```

Comment:
the new block of code should only be executed if user selected "gpt" as the "api"